layouts/partials/footer.html

```diff
@@ -1,3 +1,19 @@
+<aside class="sponsors section has-background-light">
```

Comment:
```suggestion
<aside class="partners section has-background-light">
```
Pentru că:
https://github.com/code4romania/tfsg.code4.ro/blob/da4fb0cd315c3cf23e5bc21fb7cde81fe44da350/src/scss/_apps.scss#L52-L60